pkgs/by-name/st/stats/package.nix

```diff
@@ -22,8 +22,8 @@ stdenvNoCC.mkDerivation (finalAttrs: {
   installPhase = ''
     runHook preInstall
 
-    mkdir -p $out/Applications
-    cp -r *.app $out/Applications
+    mkdir -p "$out/Applications"
+    cp -r "*.app" "$out/Applications"
```

Comment:
Are you sure this is correct? This now tries to copy a file or directory that is literally named `*.app`, which is an incredibly unconventional name. The globbing looks intentional to me.

Reply:
You're right, thank you for the review